lib/gettext_i18n_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module GettextI18nRails
-  Version = VERSION = '1.8.1'
+  Version = VERSION = '1.8.2'
```

Comment:
don't bump that

Reply:
Ah, you are absolutely right! Sorry, got my wired crossed 😄